plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/wc-admin-helper.php

```diff
@@ -225,6 +225,7 @@ public function get_store_page_test_data() {
 			array( 'https://example.com/shop/uncategorized/test/', true ), // Test case 5: URL matches product page.
 			array( '/shop/t-shirt/test/', true ), // Test case 6: URL path matches product page.
 			array( 'https://example.com/about-us/', false ), // Test case 7: URL does not match any store page.
+			array( 'https://example.com/shopping-url/', false ), // Test case 8: URL path contains aprt of 'shop' page.
 		);
```

Comment:
Also, I think it's useful to add more tests, such as:

```
array( 'https://example.com/shop/?tag=test', true )
```

and add all possible [store pages defined here](https://github.com/woocommerce/woocommerce/blob/9a9f0e7658e90fe0fc75f80c3703aaf47d16ad4e/plugins/woocommerce/src/Admin/WCAdminHelper.php#L168-L174). What do you think?

Reply:
@ilyasfoo Sounds good! I'm on it

Reply:
@ilyasfoo Updated it in [Remove terms from the test](https://github.com/woocommerce/woocommerce/pull/53120/commits/437d9d236bf9c3d481424b90e5c4721b81d48679)